src/components/TextArea/TextArea.tsx

```diff
@@ -0,0 +1,129 @@
+import * as React from 'react';
+import styles from './TextArea.st.css';
+import { TEXT_AREA_DATA_HOOK, TEXT_AREA_ERROR_DATA_HOOK } from './dataHooks';
+import { Tooltip } from '../Tooltip';
+import { ReactComponent as ErrorIcon } from '../../assets/icons/Error.svg';
+import {
+  TEXT_AREA_DISABLED,
+  TEXT_AREA_ERROR,
+  TEXT_AREA_SUCCESS,
+  TEXT_AREA_THEME,
+} from './dataKeys';
+import { TooltipSkin } from '../Tooltip/TooltipEnums';
+
+export enum TextAreaTheme {
+  Box = 'box',
+  Line = 'line',
+}
+
+export interface TextAreaProps {
+  value: string;
+  error?: boolean;
+  success?: boolean;
+  dataHook?: string;
```

Comment:
There's no need to add the `dataHook` prop unless you're using it in your tests.
In any case, if you want to leave this, so we now want to use the native `data-hook` instead of `dataHook`.
Just so it will be more like the native html components.